tests/acceptance/bs-navbar-test.js

```diff
@@ -0,0 +1,13 @@
+import { test } from 'qunit';
+import moduleForAcceptance from '../../tests/helpers/module-for-acceptance';
+
+// TODO: Write this
+moduleForAcceptance('Acceptance | bs navbar');
+
+test('visiting /navbars', function(assert) {
+  visit('/navbars');
+
+  andThen(function() {
+    assert.equal(currentURL(), '/navbars');
+  });
+});
```

Comment:
I don't think we need an acceptance test here. It should all be covered by integration tests